From 824ce3a400a452929f90a038033c83055c01a2d7 Mon Sep 17 00:00:00 2001
From: Fedora X Ninjas <x@fedoraproject.org>
Date: Mon, 24 Nov 2008 12:59:09 +1000
Subject: [PATCH] Map TLDE to Zenkaku/Hankaku in jp, and BKSL to bracket/brace right (#469537).

This patch is only needed for jp on a non-jp106 keyboard model.
---
 symbols/jp |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/symbols/jp b/symbols/jp
index 694090c..ef0b0f7 100644
--- a/symbols/jp
+++ b/symbols/jp
@@ -24,6 +24,13 @@ xkb_symbols "common" {
 	symbols[Group1]= [ Zenkaku_Hankaku, Kanji ]
     };
 
+    key <TLDE> {
+	type[Group1]="PC_ALT_LEVEL2",
+	symbols[Group1]= [ Zenkaku_Hankaku, Kanji ]
+    };
+
+    key <BKSL> { [ bracketright, braceright ] };
+
     key <AE01> { [ 1, exclam		] };
     key <AE02> { [ 2, quotedbl		] };
     key <AE03> { [ 3, numbersign	] };
-- 
1.6.0.3

